specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/longTermRetention.json

```diff
@@ -0,0 +1,723 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-03-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationName}/servers/{serverName}/databases/{databaseName}/longTermRetentionBackups/{backupName}": {
```

Comment:
longtermRetentionbackups dont live inside a resource group? This does not seem right since the server and database are both under resource group and this is a child resource under databases.

Reply:
This is because the customer can drop their database but we still need a way to get their backups for restore for up to 10 years, well beyond how long CMS will keep information about a dropped database around.

Reply:
Sounds good!